x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/signals_mapping.json

```diff
@@ -286,6 +286,13 @@
               "enabled": {
                 "type": "keyword"
               },
+              "execution": {
+                "properties": {
+                  "uuid": {
+                    "type": "keyword"
+                  }
+                }
+              },
```

Comment:
As far as I can tell it looks like [we're still using](https://github.com/elastic/kibana/blob/194d5a94ba073dbc19e1a33cd21454ac68b2dc9c/x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/create_index_route.ts#L100) the `signals_mapping` with RuleRegistry, so this change and the update to `get_signals_template.test.ts.snap` is necessary.